base/shell/cmd/lang/zh-TW.rc

```diff
@@ -505,16 +509,16 @@ RESTORE       恢復視窗\n\
     STRING_COPY_FILE "        %d 個檔案已複製\n"
     STRING_DELETE_WIPE "已擦除"
     STRING_FOR_ERROR "無效的變數指定。"
-    STRING_SCREEN_COL "無效的列值"
-    STRING_SCREEN_ROW "無效的行值"
+    STRING_SCREEN_COL "無效的行值"
+    STRING_SCREEN_ROW "無效的列值"
     STRING_TIMER_TIME "計時器 %d 是 %s："
-    STRING_MKLINK_CREATED_SYMBOLIC "已為 %s <<===>> %s 創建符號連結\n"
-    STRING_MKLINK_CREATED_HARD "已為 %s <<===>> %s 創建硬連結\n"
-    STRING_MKLINK_CREATED_JUNCTION "已為 %s <<===>> %s 創建接合點\n"
-    STRING_MORE "更多？"
-    STRING_CANCEL_BATCH_FILE "\r\nCtrl-Break 已被按下。停止批處理檔案嗎？ (Yes/No/All) "
-    STRING_INVALID_OPERAND "無效操作數。\n"
-    STRING_EXPECTED_CLOSE_PAREN "期望 ')'。\n"
-    STRING_EXPECTED_NUMBER_OR_VARIABLE "期望數值或變數名。\n"
+    STRING_MKLINK_CREATED_SYMBOLIC "已為 %s <<===>> %s 建立符號連結\n"
+    STRING_MKLINK_CREATED_HARD "已為 %s <<===>> %s 建立硬連結\n"
+    STRING_MKLINK_CREATED_JUNCTION "已為 %s <<===>> %s 建立接合點\n"
+    STRING_MORE "更多?"
```

Comment:
Hmm... why use the English version of question mark here?

Reply:
because I decided not to use Chinese version of punctuation, (maybe) until the Chinese font is shown properly in command prompt (without extra care).

Reply:
nvm I may mistakenly converted back to English form, I will change back soon.